mcli/mcli-hf-eval.yaml

```diff
@@ -1,7 +1,7 @@
 integrations:
 - integration_type: git_repo
   git_repo: mosaicml/llm-foundry
-  git_branch: v0.2.0
+  git_branch: # v0.2.0
```

Comment:
We need a tagged branch here, can't be tip of dev as that is too unstable.

Reply:
I don't think this was resolved?

Reply:
Isn't v.0.2.0 stable? We can also just comment it out entirely and let users put whatever tag/branch they want

Reply:
Yes `v0.2.0` is stable, but this diff comments it out, which means that this yaml defaults to using the tip of `dev`. I think this is OK, but we should update this to a version upon the next release. Defer to @dakinggg here.

Reply:
My preference is to have the MCLI yamls remain pinned to releases and then update them when we do a release. If we really _need_ to break this so we can give the yaml to a customer, it is ok. But the yaml should pin to a specific commit that works (e.g. https://github.com/mosaicml/llm-foundry/blob/667006889b5aab99fbecf1f927704fa7645e1799/mcli/mcli-llama2-finetune.yaml#L1-L6 for llama2)

Reply:
Should I add v.0.2.0 back in?